controller/responseTypeController.js

```diff
@@ -0,0 +1,39 @@
+const ResponseType = require('../model/responseTypeModel');
+const HttpStatus = require('http-status-codes');
+
+
+exports.getAll = function(req, res) {
+    
+    ResponseType.find().then( responseTypes => {
+        if(responseTypes) {
```

Comment:
As @SavoX2 mentioned `find` returns an array so the if statement will always be `true`. You could just return `responseTypes` without any check.